Source/Core/VideoBackends/Vulkan/VulkanContext.cpp

```diff
@@ -77,22 +79,48 @@ bool VulkanContext::CheckValidationLayerAvailablility()
   res = vkEnumerateInstanceLayerProperties(&layer_count, layer_list.data());
   ASSERT(res == VK_SUCCESS);
 
-  // Check for both VK_EXT_debug_report and VK_LAYER_LUNARG_standard_validation
-  return (std::find_if(extension_list.begin(), extension_list.end(),
-                       [](const auto& it) {
-                         return strcmp(it.extensionName, VK_EXT_DEBUG_REPORT_EXTENSION_NAME) == 0;
-                       }) != extension_list.end() &&
-          std::find_if(layer_list.begin(), layer_list.end(), [](const auto& it) {
-            return strcmp(it.layerName, "VK_LAYER_KHRONOS_validation") == 0;
-          }) != layer_list.end());
+  bool supports_validation_layers =
+      std::find_if(layer_list.begin(), layer_list.end(), [](const auto& it) {
+        return strcmp(it.layerName, VALIDATION_LAYER_NAME.c_str()) == 0;
+      }) != layer_list.end();
+
+  bool supports_debug_utils =
+      std::find_if(extension_list.begin(), extension_list.end(), [](const auto& it) {
+        return strcmp(it.extensionName, VK_EXT_DEBUG_UTILS_EXTENSION_NAME) == 0;
+      }) != extension_list.end();
+
+  if (!supports_debug_utils && supports_validation_layers)
```

Comment:
Can you add a comment along the lines of "If we didn't find debug utils support, but validation layers are supported, check if debug utils support is provided by the validation layer (see https://developer.android.com/ndk/guides/graphics/validation-layer#enable_the_debug_callback)"?